src/styles/styles.less

```diff
@@ -6,21 +6,100 @@
 
 .sb1ds {
     font-family: 'MuseoSans-500', arial, sans-serif;
+    line-height: 1.5;
 
     @media screen and (min-width: @breakpoint-lg) {
         position: relative;
         z-index: 0;
     }
+
+    b,
+    strong,
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6
+     {
+        .ffe-strong-text;
+    }
+}
+
+.sb1ds-intro {
+    &::after {
+        @media screen and (min-width: @breakpoint-lg) {
+            content: "";
+            display: block;
+            height: 320px;
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            z-index: -1;
+            background-color: @ffe-blue-cobalt;
+            background-image: url('data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20viewBox%3D%220%200%20948%2044%22%3E%3Cpath%20fill%3D%22%23FFFFFF%22%20fill-rule%3D%22evenodd%22%20d%3D%22M948.833324%2044c-156.014091%200-196.629091-28.783777-275.10464-28.783777-78.475549%200-87.754399%2010.982687-177.750909%2010.982687S386.900513%200%20292.210156%200C197.5198%200%20179.35287%2044-.67114%2044h949.504464z%22/%3E%3C/svg%3E');
+            background-repeat: no-repeat;
+            background-position: 80% 100.2%;
+            background-size: 948px 44px;
+        }
+    }
+
+    &--frontpage {
+        margin: 10px 0 20px 0;
+
+        &::after {
+            @media screen and (min-width: @breakpoint-lg) {
+                height: 380px;
+            }
+        }
+
+        @media screen and (min-width: @breakpoint-lg) {
+            height: 290px;
+        }
+    }
+
+    &--section {
+        margin: 0 0 30px 0;
+
+        @media screen and (min-width: @breakpoint-lg) {
+            height: 160px;
+        }
+    }
+
+    &__heading,
+    &__paragraph {
+        @media screen and (min-width: @breakpoint-lg) {
+            color: @ffe-white;
+        }
+    }
 }
 
 .sb1ds-main {
+    margin-top: 20px;
+
     @media screen and (min-width: @breakpoint-lg) {
-        margin-top: 70px;
+        margin-top: 90px;
     }
 }
 
 .sb1ds-section {
     margin: 30px 0;
+
+}
+
+.sb1ds-section-static {
+    margin: 0 0 50px 0;
```

Comment:
Can this be replaced with `margin-bottom` or which ever one is set? Since only one value is set I find that easier to read :)

Reply:
Done!